sig-cluster-lifecycle/README.md

```diff
@@ -102,6 +102,10 @@ The following [subprojects][subproject-definition] are owned by sig-cluster-life
 ### cluster-api-provider-gcp
 - **Owners:**
   - [kubernetes-sigs/cluster-api-provider-gcp](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/master/OWNERS)
+- **Meetings:**
+  - Cluster API Provider GCP office hours: [Wednesdays at 10:00 PT (Pacific Time)](TBD) (monthly on 1st Wednesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```

Comment:
If you are sure about this cadence just remove the TBD.

Reply:
the `TBD` i guess is the zoom URL

Reply:
we are locked outside of our Zoom account currently:
https://kubernetes.slack.com/archives/C1TU9EB9S/p1649180197245759

Reply:
thanks!

Reply:
we got our zoom acc access back, thanks to @mrbobbytables
let me create a meeting for CAPG and message you on slack about details.